src/components/Sidebar/index.tsx

```diff
@@ -20,31 +20,31 @@ const Sidebar: React.FC<ISidebar> = ({ setSidebar, sidebar }) => {
       <StyledNav>
         <ul>
           <li>
-            <div style={{ height: '22px', width: '22px' }}>
+            <div style={{ height: `${5 * GU}px`, width: `${5 * GU}px` }}>
```

Comment:
Let's remember to use the `css` prop :)

Reply:
Done! 9bff08bcf588badca595e56b8f06e8cfde1502d0